Assets/Scripts/Managers/MainMenuManager.cs

```diff
@@ -0,0 +1,22 @@
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MainMenuManager : Singleton<MainMenuManager>
+{
+    public void NewGame()
+    {
+        DataPersistenceManager.Instance.NewGame();
+        SceneManager.LoadScene("ForestTestingEthan");
+    }
+
+    public void Continue()
+    {
+        DataPersistenceManager.Instance.LoadGame();
+        SceneManager.LoadScene("ForestTestingEthan");
+    }
+
+    public void QuitGame()
+    {
+        Application.Quit();
```

Comment:
```suggestion
        Application.Quit();
        #if UNITY_EDITOR
        UnityEditor.EditorApplication.isPlaying = false;
        #endif
```
Enable the quit button to work in editor